Meta/CMake/link_libc_headers.cmake

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.22)
+
+if (NOT DEFINED SERENITY_ARCH OR NOT DEFINED SERENITY_SYSROOT)
+    message(FATAL_ERROR "SERENITY_ARCH and SERENITY_SYSROOT must be defined")
+endif()
+
+set(CMAKE_CURRENT_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/Userland/Libraries/LibC")
+set(CMAKE_STAGING_PREFIX "${SERENITY_SYSROOT}")
+set(CMAKE_INSTALL_INCLUDEDIR usr/include)
+set(LIBC_LINK_HEADERS_ONLY On)
+
+include(Userland/Libraries/LibC/CMakeLists.txt)
```

Comment:
Um. I haven't even gotten to what strange mess you've made in Libc 's CMake lists, but my gut feeling is that what you *wanted* to write is more like

```
include(Userland/Libraries/libc_headers.cmake)

link_libc_headers()
```

Reply:
If you do this, it should be possible to run this in script mode (cmake -P) without things like a minimum required(?)